src/lib/definitions.js

```diff
@@ -782,6 +782,7 @@ var createParseIntArgDefault;
             ),
             define('(document + [])[12]', HTMLDOCUMENT),
             define('escape(ARRAY_ITERATOR)[30]', NO_OLD_SAFARI_ARRAY_ITERATOR),
+            define('escape((FILTER + [][[]] + [][[]] + true + FILTER)[100])[2]', NO_V8_SRC), // *
```

Comment:
Nice. There used to be a constants `RP_9_U: 'undefined'`, but it became unused. We could readd it and write the above as

```js
define('escape((FILTER + RP_9_U + RP_9_U + RP_4_N + FILTER)[100])[2]', NO_V8_SRC)